docs/sandbox/SiriAzureUpdaters.md

```diff
@@ -0,0 +1,7 @@
+# Siri Azure Updater
+
+This is module developed by Skånetrafiken. It provides support for fetching SIRI SX / ET messages through
```

Comment:
This needs to be linked form the top level sandbox list and contain the same sections as the other sandbox features.